packages/teams-js/src/public/appWindow.ts

```diff
@@ -27,14 +33,17 @@ export interface IAppWindow {
   addEventListener(type: string, listener: Function): void;
 }
 
+/**
+ * Child app window object that can be used to send messages to and fro from parent window.
```

Comment:
>to send messages to and fro from parent window [](http://example.com/codeflow?start=44&length=46)

consider "to communicate with parent window"

Reply:
Updated in latest commit